docs/General/Manuals/client-manual.rst

```diff
@@ -1160,20 +1160,20 @@ Nations and Diplomacy View
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The :guilabel:`Nations and Diplomacy View` is actually two views accessed from the same place on the `Top
-Function Bar`_. When you click the button for it as shown in :numref:`Nations and Diplomacy View Button`, the
+Function Bar`_. When you click the button for it as shown in :numref:`Nations and Diplomacy view button`, the
 client will switch to a list of nations that you are playing against in a table format.
 
-.. _Nations and Diplomacy View Button:
+.. _Nations and Diplomacy view button:
 .. figure:: ../../_static/images/gui-elements/top-bar-nations.png
   :align: center
   :scale: 50%
   :alt: Freeciv21 Nations
   :figclass: align-center
 
-  Nations and Diplomacy View Button
+  Nations and Diplomacy view button
 
 
-:numref:`Nations and Diplomacy View Button with Diplomacy View Open` gives a sample of the
+:numref:`Nations and Diplomacy view button with Diplomacy View open` gives a sample of the
```

Comment:
Here as well, it's called "Nations and Diplomacy View", so "View" should be capitalized